src/ui/MLFacet/MLFacet.ts

```diff
@@ -248,6 +248,30 @@ export class MLFacet extends Component {
     this.handleFacetValuesChanged();
   }
 
+  /**
+   * Requests an additional amount of values equal to the [`number of values`]{@link MLFacet.options.numberOfValues}.
+   *
+   * Does trigger a query automatically.
+   */
+  public showMoreValues(): void {
+    this.ensureDom();
+    this.logger.info('Show more values');
+    this.mLFacetQueryController.increaseNumberOfValuesToRequest();
+    this.triggerNewQuery();
```

Comment:
It might make sense to support, from a "programmatic" point of view, a method to `showMoreValues(numberToAdd)` or something like that, and have the "default" value be the `numberOfValuesToRequest`.

Even if the "show more click" would always call it with the default value, it might be nice for clients interacting with the facet through code to be able to control/override the behaviour of the expand if they wish to.

Reply:
I understand, i'll add it